release-1.7/release-1.7.md

```diff
@@ -0,0 +1,58 @@
+# Proposed timeline for 1.7
+We will begin the 1.7 release the first week of April 2017.
+
+The proposal below is identical in layout to the 1.5 and 1.6 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+
+## 1.7 Release Schedule
+- April 3 (Mon) coding start (7w)
+- April 26 (Weds) features repo freeze
+- May 17th (Weds) v1.7.0-alpha.1
+- May 24th (Weds) v1.7.0-alpha.2
+- May 31st (Weds) Start code freeze; Cut release-1.7 branch and v1.7.0-beta.0; Feature Burndown Meetings begin
+- June 13st (Weds) v1.7.0-beta
+- June 20st (Weds) v1.7.release-candidate
+- June 28th (Weds) - v1.7.0 (release!)
+
+## 1.7 Details
+
+### April 3 - May 31
+- 8 week coding period
+- Release 1.7 alphas every 2 weeks
```

Comment:
I actually think we could decrease the two week interval, since it tend to slip anyway.
I mean, if we have a target like every 10th day during the cycle, we could actually be able to release biweekly ;)